### DISCUSSION TITLE
Strategy for client only

### DISCUSSION THREAD

**Discussion by yaroz** [rustdesk/rustdesk#6668] [General]
In Anydesk, we have the option to create a client only, where it can only receive connection requests, not make them.. and they cannot make config changes.  Basically, if we want to make a config change, we have to create a new installer and reinstall to the workstation.  The client only would look like the qs one that @rustdesk  proposed in https://github.com/rustdesk/rustdesk/discussions/5618#discussioncomment-7284221.  I know qs is low in the priority list, but figured when you get that working, you can add a tick box to the strategies for incoming connections only or something similar.  

![Screenshot 2023-12-12 145105](https://github.com/rustdesk/rustdesk/assets/20463573/831b8c64-d741-4341-ae82-e5c3d9e3d8a1)

**Comment by 1cue-jrupp** [comment]
We'd also really like this feature for a migration to the rustdesk payed plans!

**Comment by rustdesk** [comment]
I start to work on this today, including [custom client generator](https://github.com/rustdesk/rustdesk/discussions/6507). You will see it in one or two weeks. Apologize for the delay because of our full todo list.

some note:
> move /y newicon.ico rustdesk.exe
> modify plist etc https://gist.github.com/syneart/4a8724cd479d31f0f742f499f807dcb2

**Reply by Sebshino** [reply]
hello, I'm really looking forward to seeing this tool, hoping it won't be detected as malicious.  The fact that the current client is detected malicious is a big obstacle for some clients.... we have to go through anydesk to be able to install rusdesk.....

**Reply by bilogic** [reply]
Can we have a quick support that must get UAC or terminate? https://github.com/rustdesk/rustdesk/discussions/7882

**Comment by Sebshino** [comment]
hello, can you give us an update on rustdesk quick support?

**Comment by rustdesk** [comment]
## Update

https://twitter.com/rustdesk/status/1767588052325572682

**Comment by rustdesk** [comment]
Update: https://twitter.com/rustdesk/status/1769171628426944539

**Reply by Sebshino** [reply]
the quicksupport works flawlessly, however if I understood correctly it is that the quicksupport is generated on your servers?

**Reply by Sebshino** [reply]
unfortunately the executables are detected as viruses by antiviruses......

**Reply by pdifeo** [reply]
I think it is related to signing certificate problem. About I'm waiting for 1.2.4 stable version.

**Reply by C-Duv** [reply]
Are the Linux and macOS versions of this "QuickSupport" client software planned too?

**Reply by rustdesk** [reply]
https://twitter.com/rustdesk/status/1782250852574769663

**Reply by C-Duv** [reply]
Sorry but I fail to see how this third-party publication answers my question about Linux and macOS software releases.

**Comment by yaroz** [comment]
@rustdesk Trying to set a password and assign the client to a user on the custom client, using "--password <password>" and "--assign --token <token> --user_name <username>" and it's not working like it does with the standalone client.  Is it meant to work this way?

**Reply by rustdesk** [reply]
We changed nothing here, should work.

**Reply by yaroz** [reply]
Disregard.. the config wasn't working, because I was pointing to a different server config, and the password didn't work because I wasn't in the administrator command window.  Thanks for doing that.

**Comment by HansRoulade** [comment]
@rustdesk I'm voting for a community-usable version of individualized clients. Following the build guide, I suggest commandline options for connection direction types as well as disabling installation/settings/address book/TCP listen port/user account/preset password if I'm willing to go the extra mile to compile my own client. Personalizing options like integration of own icon and logo as well as the convenience of the client generator GUI as it already exists could still be considered Pro features nevertheless.

**Reply by rustdesk** [reply]
No, Pro only
